### ISSUE TITLE
Socket problems

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#7680]
Hi suddenly my socket isnt displaying onliine user names it just shows a count. This was working fine until last night.

I have changed PHP versions.
Installed a fresh test site (still same problem)
Rebooted the server
Nothing shows in the logs.

Any ides what is causing this?

![Screenshot 2023-02-17 at 12 35 18](https://user-images.githubusercontent.com/107706817/219655617-03722246-9a0f-45f9-9d2c-da1bb39f5fb0.png)

**Comment by DanielnetoDotCom** [comment]
that is correct. the new socket has some huge performance updates. the user's list is disabled now but may come back in the future.

**Comment by para2022** [comment]
Oh thats a crying shame as that was so useful seeing who was active on the site and logged in. But thanks for the heads up mate ive been working on fixing this for the past 6 hours totally unaware you had changed it.
